File: lib/Responsavel.dart
class Responsavel {
  String _id;
  String _nome;
  String _email;
  String _senha;
  String _telefone;

  Responsavel(this._id, this._nome, this._email, this._senha, this._telefone);

  String get telefone => _telefone;

  set telefone(String value) {
    _telefone = value;
  }

  String get senha => _senha;

  set senha(String value) {
    _senha = value;
  }

  String get email => _email;

  set email(String value) {
    _email = value;
  }

  String get nome => _nome;

  set nome(String value) {
    _nome = value;
  }

  String get id => _id;

  set id(String value) {
    _id = value;
  }

}

File: responsavel.php
<?php
    $servername = "localhost";
    $username = "root";
    $password = "";
    $dbname = "pdm";
    $table = "responsavel";

    //$action = $_GET['acao'];

    // Create connection
    $conn = new mysqli($servername, $username, $password, $dbname);
    // Check connection
    if ($conn->connect_error) {
        die("Connection failed: " . $conn->connect_error);
        return;
    } else {
        echo "ok";

    }


    $nome = $_POST["nome"];
    $email = $_POST["email"];
    $senha = $_POST["senha"];
    $telefone = $_POST["telefone"];

    $query = "insert into responsavel (id, nome, email, senha, telefone) values (null, '$nome', '$email', '$senha', '$telefone')";
    $result = mysqli_query($conn, $query);

    if($result){
        echo "ok";
    } else {
        echo "error";
    }

?>
